README.md

```diff
@@ -3,11 +3,13 @@
 
 # Chasqui
 
-Chasqui is a simple, lightweight, persistent implementation of the publish-subscribe (pub-sub)
-messaging pattern for service oriented architectures.
+Chasqui is a simple, lightweight, persistent implementation of the
+publish-subscribe (pub-sub) messaging pattern for service oriented
+architectures.
```

Comment:
@jbgo yeah, thought about this some more last night.  I think the biggest reason to use chasqui is that it is **persistent**.  I actually don't think it is the simplest way to do something like this.  Using a fully in-memory setup like delayed job that wouldn't even need redis would be the simplest (but not very helpful if you needed cross app/service events).  Using redis' native pub/sub would also be simpler but that is not persistent (at least last time I checked.

This is where chasqui can shine - you get cross app/service events (all those positives you mention in the README) and all this is **persistent** so if services go down or whatever, your queues are just backed up waiting for them to come back.  There is some "non simple" overhead for getting this benefit though (you have to setup redis, you have to run a broker, etc).